src/HtmlOptions/RequiredHtmlOptions.php

```diff
@@ -5,20 +5,31 @@
 namespace Yiisoft\Form\HtmlOptions;
 
 use Yiisoft\Validator\Rule\Required;
+use Yiisoft\Validator\RuleInterface;
 
-class RequiredHtmlOptions implements HtmlOptionsProvider
+class RequiredHtmlOptions implements HtmlOptionsProvider, RuleInterface
 {
-    private Required $validator;
+    use ValidatorAwareTrait;
+
+    private bool $ariaAttribute = false;
 
     public function __construct(Required $validator)
     {
         $this->validator = $validator;
     }
 
+    public function withAriaAttribute(bool $value): self
```

Comment:
Is there a reason not to have aria attribute? If there is, default should be `true` anyway.

Reply:
http://html5doctor.com/on-html-belts-and-aria-braces/
> The answer is maybe for a subset of elements, but increasingly no.

Reply:
Is this conversation closed?

Reply:
Yes.